modules/programs/newsboat.nix

```diff
@@ -107,7 +107,7 @@ in {
         type = types.attrsOf types.str;
         default = { };
         example = { "foo" = ''rssurl =~ "example.com"''; };
-        description = "A list of queries to use.";
+        description = "A list of queries to use. This option won't work if <varname>urls</varname> is empty.";
```

Comment:
I guess queries don't make much sense unless you have urls to query? If so perhaps best to use an assertion? Something like

```
assertions = [{
  assertion = cfg.queries != [] -> cfg.urls != [];
  message = "Cannot specify queries unless you have urls";
}]
```

perhaps.